### PR TITLE
Update for new implement

### PR DESCRIPTION
**Pull request by zarda**
Update for #208, to adapt to a new implement in [#771](https://github.com/CARTAvis/carta-backend/pull/771).

### CODE REVIEW

File: src/test/CASA_REGION_IMPORT_EXPORT.test.ts [hunk 1]
```diff
@@ -181,15 +188,15 @@ let assertItem: AssertItem = {
                 lengthOfRegions: 7,
```
Comment (acdo2002) [inline src/test/CASA_REGION_IMPORT_EXPORT.test.ts:188]: should be lengthOfRegions: 8,

File: src/test/CASA_REGION_IMPORT_EXPORT.test.ts [hunk 2]
```diff
@@ -181,15 +188,15 @@ let assertItem: AssertItem = {
...
                 assertRegionId: {
                     index: 6,
-                    id: 14,
+                    id: 15,
                 },
             },
             {
                 success: true,
                 lengthOfRegions: 7,
```
Comment (acdo2002) [inline src/test/CASA_REGION_IMPORT_EXPORT.test.ts:196]: should be lengthOfRegions: 8,

File: src/test/CASA_REGION_IMPORT_EXPORT.test.ts [hunk 3]
```diff
@@ -181,15 +188,15 @@ let assertItem: AssertItem = {
...
                 assertRegionId: {
                     index: 6,
-                    id: 21,
+                    id: 22,
```
Comment (acdo2002) [inline src/test/CASA_REGION_IMPORT_EXPORT.test.ts:199]: should be id: 23,

**Review by acdo2002 — CHANGES_REQUESTED** [review]
expect(importRegionAckProperties[importRegionAckProperties.length - 1]).toEqual(String(Region.assertRegionId.id));

should be
expect(importRegionAckProperties[Region.assertRegionId.index]).toEqual(String(Region.assertRegionId.id));

File: src/test/CASA_REGION_IMPORT_EXPORT.test.ts (hunk 1 above)
Comment (zarda) [reply src/test/CASA_REGION_IMPORT_EXPORT.test.ts]: Correct

File: src/test/CASA_REGION_IMPORT_EXPORT.test.ts (hunk 2 above)
Comment (zarda) [reply src/test/CASA_REGION_IMPORT_EXPORT.test.ts]: correct

File: src/test/CASA_REGION_IMPORT_EXPORT.test.ts (hunk 3 above)
Comment (zarda) [reply src/test/CASA_REGION_IMPORT_EXPORT.test.ts]: got it

**Comment by zarda** [comment]
@ajm The demo building for branch 610_line_region went wrong so this testing will fail at the moment. Thought this testing could pass on a local instance.

**Comment by ajm-asiaa** [comment]
@zarda It seems to be OK now. Works fine on MacOS against the `pam/610_line_region` backend branch.

**Pull request merged**